css/style.css

```diff
@@ -24,8 +24,52 @@ button a:hover{
   text-decoration: none;  
 }
 
+/*Stying for Tables made with div tags instead of table tags
+Adapted from: 
+http://www.dummies.com/web-design-development/css3/using-the-div-tag-to-create-tables/
+
+and from CSS and HTML generated at: 
+http://divtable.com/generator/
+*/
+.Table {
```

Comment:
On a separate note, to reduce the number of requests made to the server, should I also combine style.min.css and bootstrap.min.css into one CSS file?